test/test_ref_eager.py

```diff
@@ -0,0 +1,556 @@
+from __future__ import annotations
+
+import contextlib
+import functools
+import io
+import math
+import os
+import unittest
+
+import torch
+
+from . import test_dot
+from . import test_examples
+from . import test_grid
+from . import test_loops
```

Comment:
In this test file, we directly import these test classes for ref eager mode testing, and skip the individual tests that are not supported yet.

Reply:
I suspect this won't work in fbcode.  Perhaps we should have a separate test shard for reference mode rather than duplicating the tests here.

That will also mean new tests are checked in ref mode automatically.